docs/processors/conditions.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Conditional Execution'
+sidebar_position: 3
+---
+
+When a [processor](/docs/connectors/getting-started) is attached to a connector or to a pipeline, we may still want to specify conditions for its execution. To do this, we simply add a `condition` key to the processor definition in our [Pipeline Configuration File](/docs/pipeline-configuration-files).
```

Comment:
maybe we should mention the other option to set the `Condition` at the end of this paragraph, just a sentence saying that you can also set the condition using the HTTP API https://conditional-processors.conduit-site.pages.dev/api/#post-/v1/processors
WDYT? @lovromazgon @raulb

Reply:
@maha-hajja I think you're right. I should at least mention this option exists. Will do that. Thanks for the suggestion

Reply:
@maha-hajja I included a mention to HTTP and provided better links https://github.com/ConduitIO/conduit-site/pull/65/commits/ac5aafbf05ef827c80539099d439c83794eacc31. 

Going to ship this PR, but please let me know if you think I should make a change and will address it separately. Thanks for reviewing!

Reply:
looks good!